r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -341,10 +337,9 @@ test_that("median()", {
     tbl,
     warning = "median\\(\\) currently returns an approximate median in Arrow"
   )
-  local_edition(3)
-})
+}, classes = "arrow.median.approximate"))
```

Comment:
OOOH, that's fantastic. I don't think I knew that `suppressWarnings()` would let you specify classes, that's really nice!

Reply:
Yeah, I think this was an R 4.0 thing. Ironically I was playing around with similar ideas at the same time in [rror](https://github.com/alistaire47/rror). Maybe there's still some value to that package, but I suspect it's mostly obsolete now. I should really write a blog post about the concepts, though; nobody classes conditions enough